artifacthub/library/pod-security-policy/forbidden-sysctls/1.0.0/template.yaml

```diff
@@ -33,12 +42,20 @@ spec:
       rego: |
         package k8spspforbiddensysctls
 
+        # Block if forbidden
         violation[{"msg": msg, "details": {}}] {
             sysctl := input.review.object.spec.securityContext.sysctls[_].name
             forbidden_sysctl(sysctl)
             msg := sprintf("The sysctl %v is not allowed, pod: %v. Forbidden sysctls: %v", [sysctl, input.review.object.metadata.name, input.parameters.forbiddenSysctls])
         }
 
+        # Block if not explicitly allowed
+        violation[{"msg": msg, "details": {}}] {
+            sysctl := input.review.object.spec.securityContext.sysctls[_].name
+            not allowed_sysctl(sysctl)
+            msg := sprintf("The sysctl %v is not explictly allowed, pod: %v. Allowed sysctls: %v", [sysctl, input.review.object.metadata.name, input.parameters.allowedSysctls])
```

Comment:
We should add a comment that `input.parameters.allowedSysctls` in `sprintf()` is load-bearing.

Without that line, this change would be backwards-incompatible and that seems non-obvious. Adding a comment will help prevent accidentally breaking this requirement in the future.

Reply:
Sorry, what specifically do you mean by "load-bearing"?